server/api/users.js

```diff
@@ -91,14 +116,13 @@ router.post('/users/new', apiLimiter, multer.single('avatar'), async (req, res,
       referrer,
       locale = 'en',
       accessToken,
+      platformUserId,
       secret,
+      isEmailVerified,
```

Comment:
should not trust `isEmailVerified` directly from user, should only trust when 
1. oauth provider provides a trusted email
2. oauth provider email == user input email